packages/block-library/src/tag-cloud/style.scss

```diff
@@ -18,4 +18,18 @@
 		margin-left: 5px;
 		text-decoration: none;
 	}
+
+	&.is-style-pill a {
```

Comment:
Should we remove the underline in this case?

Reply:
Yeah, I don't think an underline is necessary.